app/i18n/cz/install.php

```diff
@@ -4,7 +4,9 @@
 	'action' => array(
 		'finish' => 'Dokončit instalaci',
 		'fix_errors_before' => 'Chyby prosím před přechodem na další krok opravte.',
+		'keep_install' => 'Keep previous installation',  // TODO: translate
```

Comment:
I don't know how :-( I still struggle with github. Do I have to fork "FreshRSS/FreshRSS" again? Switch to some other branch (how? - git is in my 'personal fork'). Sorry, it seems that  I just can't grasp the git fundamentals :-(
"git status" on localhost gives me:
On branch 814-i18n-czech-translation
Your branch is up-to-date with 'origin/814-i18n-czech-translation'.
Again, I'm very sorry. Thanks for any help.